source/_docs/clamav.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Enabling ClamAV for WordPress or Drupal
+description: Learn how to protect your site against malicious files with malware and other potential threats using ClamAV, an open source antivirus engine.
+tags: [clamav, security, files]
+categories: [security]
+---
+
+## What is ClamAV?
+
+**ClamAV** is an open source antivirus engine for detecting trojans, viruses, malware & other malicious threats. It supports multiple file formats, file and archive unpacking, and multiple signature languages.
+
+A file that may look like a normal JPG or PDF file may actually be malware that run scripts within your site. They may come from files that have been uploaded via fields, attachments or other media forms. These may end up in your public files directory and might cause alerts, harm site SEO scores, compromise data, and or lower site credibility.
```

Comment:
does WordPress/Drupal refuse to upload files that are malicious?

Reply:
what happens when ClamAV detects a mailicious file? It just logs? or prevents upload?

Reply:
Oh, that depends on a config setting, which defaults to "block" - at least in the D8 module. Haven't got the WP plugin to work yet.